DOCUMENTATION.md

```diff
@@ -216,10 +216,17 @@ You can leave out this argument if your service only has one reply-to email addr
 
 To send a file by email, add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file.
 
-The file will be available for the recipient to download for 18 months.
-
 The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file.
 
+Your file will be available to download for a default period of 78 weeks (18 months). From 29 March 2023 we will reduce this to 26 weeks (6 months) for all new files. Files sent before 29 March are not affected.
+
+To help protect your files you can also:
+
+* ask recipients to confirm their email address before downloading
+* choose the length of time that a file is available to download
+
+To turn these features on or off, you will need to update your API client to the latest version.
```

Comment:
I would suggest we remove this sentence for two reasons

1. We don't have this precedent anywhere else of telling people they need to update their API client to take advantage of a new feature
2. At some point it means we need to remove this line as it won't be true (as users by that point may already be using a version of the client that contains this functionality)

Reply:
Alternatively we say that they need to update to at least a specific client version x.y.z.